shared/desktop/app/store-helper.js

```diff
@@ -1,10 +1,10 @@
 // @flow
 import {ipcMain} from 'electron'
-import {selector as menubarSelector} from '../../menubar'
-import {selector as pineentrySelector} from '../../pinentry'
-import {selector as remotePurgeMessageSelector} from '../../pgp/container.desktop'
-import {selector as trackerSelector} from '../../tracker'
-import {selector as unlockFoldersSelector} from '../../unlock-folders'
+import {selector as menubarSelector} from '../../menubar/selector'
```

Comment:
split the selector out of the react components. this eliminates pulling in react into the main renderer